matlab/+BalanceLaws/compute_numerical_solution.m

```diff
@@ -4,12 +4,18 @@
 
 % Takes the initialized fields and advances the solution in time
 
-global I_Mesh I_TI I_BalanceLaws I_Tech I_RunOps I_ConservedVars
+global I_Mesh I_TI I_BalanceLaws I_Tech I_RunOps I_Results
 
 if strcmp(I_Tech('REAL'),'float')
     current_time = single(zeros(2));
+    components = single(zeros(2));
+    norm2_output = single(zeros(1, I_Tech('num_groups')));
+    error = single(zeros(I_BalanceLaws('NUM_CONSERVED_VARS'), I_TI('num_steps')+1));
```

Comment:
Since `error` is a Matlab function and this array stores the error values for every time step, we might want to change the name. But this is up to you.